client/rust/src/lib.rs

```diff
@@ -696,15 +696,28 @@ mod test {
         // Bid for 10 SOL
         market_value.market_expand().unwrap();
 
-        let quote_global_trade_account: GlobalTradeAccounts = GlobalTradeAccounts {
+        let mut lamports: u64 = 100_000;
```

Comment:
different than the extra rent required to pay?

Reply:
There is no extra rent.
This is so there are enough lamports for the gas prepay. The test setup assumes there is an order already which means there is a deposit available.